layouts/shortcodes/katex.html

```diff
@@ -0,0 +1,15 @@
+{{ if not (.Page.Scratch.Get "katex") }}
+  <!-- Include katex only first time -->
+  <link
+    rel="stylesheet"
+    href="{{ index (index .Site.Data.assets "katex.css") "src" | relURL }}"
+  />
+  <script defer src="{{ index (index .Site.Data.assets "katex.js") "src" | relURL }}"></script>
+  {{ .Page.Scratch.Set "katex" true }}
+{{ end }}
+
+<span class="gdoc-katex">
```

Comment:
Please rename the class to `gblog-katex`.

Reply:
Totally missed that, sorry. Should be fixed now.